src/components/Layout/AppBar.tsx

```diff
@@ -1,41 +1,49 @@
-// App bar at the bottom of page
-// usages: everywhere
 'use client';
 
-import Link from "next/link";
-import LinkIcon from "next/link";
+import HomeRoundedIcon from '@mui/icons-material/HomeRounded';
+import GroupsRoundedIcon from '@mui/icons-material/GroupsRounded';
+import EqualizerRoundedIcon from '@mui/icons-material/EqualizerRounded';
+import GridViewRoundedIcon from '@mui/icons-material/GridViewRounded';
+import CarpenterRoundedIcon from '@mui/icons-material/CarpenterRounded';
+import {BottomNavigation, BottomNavigationAction} from "@mui/material";
+import Paper from '@mui/material/Paper';
+import { usePathname } from 'next/navigation';
+import React from "react";
 import clsx from "clsx";
-import { usePathname } from "next/navigation";
-import nav from "react";
 
 //Probably could have this populate from the DB but its static so...
 const links = [
-    { name: 'Home', href: '/home', icon: 'HomeIcon' },
-    { name: 'Meta Hub', href: '/meta', icon: 'MetaIcon' },
-    { name: 'Creators', href: '/creators', icon: 'CreatorsIcon' },
-    { name: 'TTK Tool', href: '/ttk', icon: 'TtkToolIcon' },
-    { name: 'Hub', href: '/hub', icon: 'HubIcon' },
+    { name: 'Home', href: '/', icon: HomeRoundedIcon },
+    { name: 'Meta Hub', href: '/meta', icon: CarpenterRoundedIcon }, // There's no icon in MUI for a gun :P
```

Comment:
ahh guessing no gun in there XD

Reply:
Nah, felt like you'd appreciate a saw as replacement haha